bftengine/src/bftengine/PersistentStorageImp.cpp

```diff
@@ -905,6 +914,20 @@ bool PersistentStorageImp::nonExecSetIsAllowed() {
   return setIsAllowed() &&
          (!hasDescriptorOfLastExecution() || descriptorOfLastExecution_.executedSeqNum <= lastExecutedSeqNum_);
 }
+void PersistentStorageImp::setEraseMetadataStorageFlag() {
+  bool eraseMtOnStartUp = true;
```

Comment:
spell out whatever Mt is ? Presumably metadata :)

Reply:
fixed